app/controllers/api/flashcards_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::FlashcardsController < ApplicationController
+  MAX_AMOUNT_OF_FLASH_CARDS_TO_DISPLAY = 10
+
+  def index
+    flash_cards = Flashcard.first(MAX_AMOUNT_OF_FLASH_CARDS_TO_DISPLAY)
```

Comment:
I would go with `FlashCard` as class name

Reply:
I was asked to change it from `FlashCard` to `Flashcard`